### DISCUSSION TITLE
Loadbalancer Service

### DISCUSSION THREAD

**Discussion by sumitsimplex** [strimzi/strimzi-kafka-operator#8706] [Q&A]
why do I get two public IPs created when I enable external access using loadbalancer?

one on external-bootstrap and another on kafka-0

**Comment by scholzj** [comment]
Thsi is because of how Kafka and its discovery mechanism work. The clients need a direct access to each broker. You can check this old blog post series I wrote for more details: https://strimzi.io/blog/2019/04/17/accessing-kafka-part-1/

If you want, you can disable the bootstrap loadbalancer. But you will always need a load balancer per broker.
